spec/index.html

```diff
@@ -41,10 +41,13 @@
 
   <style>
     figure { text-align: center; }
-    table { border-collapse:collapse; }
-    table, td, th { border:1px solid black; }
+    table { border-collapse: collapse; }
+    table, td, th { border: 1px solid black; }
     caption { font-weight: bold; text-align: left ; }
-    code {color: #ff4500;}  /* Old W3C Style */
+    code { color: #ff4500; }  /* Old W3C Style */
+    @media (max-width: 767px) {
+        table { word-break: break-all; max-width: 80vw; padding: 0.2em 0.5em }
```

Comment:
I'm unclear what our overall policy is. What I think we want is to avoid breaks within words. Is that what this does?

Reply:
[No](https://developer.mozilla.org/en-US/docs/Web/CSS/word-break). 

I also think we want to avoid breaks within words, so I *think* we want this:
```suggestion
        table { word-break: normal; overflow-wrap: normal; max-width: 80vw; padding: 0.2em 0.5em }
```